tools/scripts/generate_changelog.py

```diff
@@ -0,0 +1,62 @@
+"""
+Generate changelog for github release from the changelog.rst file
+Example of execution:
+
+python tools/scripts/generate_changelog.py -v 1.15.0 -f docs/changelog.rst
+
+It outputs the result to the stdout
+"""
+
+import argparse
+import re
+
+
+def generate_changelog(version: str, path: str) -> str:
+    change_regex = r'\*\ :(feature|bug):`(\d*|-)`\ ([\w| |.]+)'
+
+    changes = {'features': [], 'bugs': []}
+    github_link = '- [{number}](https://github.com/rotki/rotki/issues/{number}) {text}'
+    not_listed_change = '- {text}'
+
+    with open(path) as f:
+        lines = f.readlines()
+        start = False
+        for line in lines:
+            if version in line:
+                start = True
+                continue
+            if start:
+                r = re.search(change_regex, line)
+                if r is not None:
+                    groups = r.groups()
+
+                    if groups[1].isdigit():
+                        text = github_link.format(number=groups[1], text=groups[2])
+                    else:
+                        text = not_listed_change.format(text=groups[2])
```

Comment:
You probably need to add an exception here for the "Added support for the following tokens" entries.
![2021-05-04_16-34](https://user-images.githubusercontent.com/1658405/117020315-9ebc3980-acf6-11eb-8266-37c992ea2a81.png)